php/Terminus/Models/Environment.php

```diff
@@ -110,6 +110,27 @@ public function backupUrl($bucket, $element) {
   }
 
   /**
+   * Changes connection mode
+   *
+   * @param [string] $value Connection mode, "git" or "sftp"
+   * @return [Workflow] $workflow
+   */
+  public function changeConnectionMode($value) {
+    switch ($value) {
+      case 'git':
+        $workflow_name = 'enable_git_mode';
+        break;
+      case 'sftp':
+        $workflow_name = 'enable_on_server_development';
+        break;
+    }
+
+    $params   = array('environment' => $this->get('id'));
```

Comment:
Should we be checking here what the current mode is? That would prevent unnecessary work if it's already in the correct mode. Downside: it would take away the ability to 'force' a mode change if something is out of whack (not sure if this is a real use-case)